web/docs/contributing.md

```diff
@@ -4,9 +4,14 @@ sidebar_label: Contributing
 slug: /contributing
 ---
 
-Any way you want to contribute is a good way, and we are happy to meet you :)!
+Any way you want to contribute is a good way, and we are happy to meet you! A single entry point for all contributors is the [CONTRIBUTING.md](https://github.com/wasp-lang/wasp/blob/main/CONTRIBUTING.md) file in our Github repo. All the requirements and instructions are there. So please check [CONTRIBUTING.md](https://github.com/wasp-lang/wasp/blob/main/CONTRIBUTING.md) for more details.
 
-Some typical ways to contribute:
-1. Join us on discord [![Discord](https://img.shields.io/discord/686873244791210014?label=chat%20on%20discord)](https://discord.gg/rzdnErX) and let's talk! We can discuss language design, new/existing features, weather, or you can just tell us how you feel about Wasp :).
-2. If there is something in docs that you think could be improved or clarified, go to [docs Github repo](https://github.com/wasp-lang/wasp) and make an issue/PR! Or, you can do it directly from here by clicking on "edit this page" at the bottom of each page.
-3. Create an issue/PR on [core Wasp repo](https://github.com/wasp-lang/wasp) to contribute directly to the language and/or compiler! Check [wapsc README](https://github.com/wasp-lang/wasp/tree/main/waspc) for more details.
+Some side notes to make your journey easier:
+
+1. Join us on discord [![Discord](https://img.shields.io/discord/686873244791210014?label=chat%20on%20discord)](https://discord.gg/rzdnErX) and let's talk! We can discuss language design, new/existing features, and weather, or you can tell us how you feel about Wasp :).
+
+2. Wasp's compiler is built with Haskell. So you'll need to be somewhat familiar with this language if you'd like to contribute to the compiler itself. If you don't have Haskell knowledge (or any dev experience at all) - no problem. There are a lot of JS-related tasks and documentation updates as well!
+
+3. If there's something you'd like to bring to our attention, go to [docs Github repo](https://github.com/wasp-lang/wasp) and make an issue/PR!
```

Comment:
```suggestion
3. If there's something you'd like to bring to our attention, go to [docs GitHub repo](https://github.com/wasp-lang/wasp) and make an issue/PR!
```
This was strange to me at first, but GitHub is how it's actually correctly capitalized. Not really important ofc, but just mentioning it since I came across it here.